iothub_client/inc/iothub_message.h

```diff
@@ -42,6 +42,12 @@ IOTHUBMESSAGE_UNKNOWN \
 */
 MU_DEFINE_ENUM(IOTHUBMESSAGE_CONTENT_TYPE, IOTHUBMESSAGE_CONTENT_TYPE_VALUES);
 
+#define IOTHUB_MESSAGE_CATEGORY_VALUE   \
+    MESSAGE_CATEGORY_TELEMETRY,         \
```

Comment:
>MESSAGE_CATEGORY_TELEMETRY [](start = 4, length = 26)

I think we need to nail down the interaction between distributed tracing and security client at some point. I will try to fix this in E2E  diagnostics branch and send you a PR.